tests/Microsoft.Bot.Builder.Prompts.Tests/ChoicePromptTests.cs

```diff
@@ -0,0 +1,334 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is a great new set of tests. I'm not clear why it's in a PR about changing IList to IEnumerable. 

Please put these tests in their own PR.

Reply:
I don't know why these are showing up as a change I'm making. These are from my pulling in recent changes from main. However, that will be moot soon.